src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const listItem = document.querySelectorAll('#thumbs a');
+
+listItem.forEach((thumb) => {
+  thumb.addEventListener('click', () => {
+    const newSrc = thumb.getAttribute('href');
+
+    largeImg.src = newSrc;
+  });
```

Comment:
You should prevent the default behavior of the click event on the thumbnail links. This can be done by calling event.preventDefault() inside the event listener callback. This will stop the browser from following the link and causing a page refresh. Here is an example of how you can do this: 
javascript
thumb.addEventListener('click', (event) => {
  event.preventDefault();
  const newSrc = thumb.getAttribute('href');
  largeImg.src = newSrc;
});
```